packages/react-magma-dom/src/components/Dropdown/DropdownMenuItem.tsx

```diff
@@ -36,12 +36,30 @@ export interface DropdownMenuItemProps
   value?: string | number;
 }
 
+export function menuColors(props) {
+  if (props.isInverse) {
+    return props.theme.colors.neutral100;
+  }
+  if (props.disabled) {
+    props.theme.colors.disabledText;
+  }
+  return props.theme.colors.neutral700;
+}
+
+export function menuBackground(props) {
+  if (props.isInverse) {
+    return props.theme.colors.primary600;
+  }
+  if (props.disabled) {
+    ('none');
```

Comment:
```suggestion
    return 'none';
```

Reply:
Again, disabled should be the first check here

Reply:
Removed.

Reply:
Looks like the return statement for the disabled case still needs to be updated